app/_how-tos/docker-compose.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Install {{site.base_gateway}} using Docker Compose
+description: Use Docker Compose to install {{site.base_gateway}}
+content_type: how_to
+permalink: /gateway/install/docker
+related_resources:
+  - text: Installation documentation
```

Comment:
```suggestion
  - text: Install {{site.base_gateway}} on a supported platform
```
Something like this? "Installation documentation" is too vague, and unspecific to the product.